### DISCUSSION TITLE
By default components are created using table in my demo project but the demo uses flex property to create components

### DISCUSSION THREAD

**Discussion by shomeel** [GrapesJS/grapesjs#5163] [Q&A]
I need help with something regarding the grapes.js. I have created a project and whenever I put a component in Canvas, every component has display : table property by default. However I have tried in grapes.js official demo, and every element has display : flex by default in their demo. What could be the issue and why the local project has different properties by default?

**Comment by artf** [comment]
You can see the demo code [here](https://github.com/GrapesJS/grapesjs/blob/gh-pages/demo.html) with all the configurations
